### ISSUE TITLE
[TSDoc> Search]: Screen reader is not announcing 'to select', 'to navigate' and 'to close' keys information which is provided on the search dialog.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/rushstack-websites#147]
**Test Environment:**
OS Version: 22H2 (OS Build 25370.1)
Browser: Edge Dev (Version 115.0.1866.1 (Official build) dev (64-bit))
URL: https://tsdoc.org/
Screen reader: Narrator

**Repro Steps:**

Open the above URL in edge dev browser and login with v-.
Turn on the narrator through Ctrl+ windows+ Enter key.
Navigate to the 'Search' button and activate it.
Navigate to the search box and type any character observe the issue.
Observe the issue.

**Actual Result:**
Screen reader is not announcing 'to select', 'to navigate' and 'to close' keys information which is provided on the search dialog.

**Expected Result:**
Screen reader should announce the  'to select', 'to navigate' and 'to close' keys information when focus is on the search box which is provided on the search dialog.
Screen reader should announce as 'Use Enter key to select or use up/down arrow key to navigate and Esc key to close' when search sugeestion appears to the user on typing text in search box.

**User Impact:**
Screen reader user will get impacted as user will not get the information which is displayed on the dialog which create difficulty in navigating or in interacting with the content.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

https://github.com/microsoft/rushstack-websites/assets/104483005/00dc37eb-0624-4937-9ee6-0d26ecb3a9b3
